whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/ObjectData.java

```diff
@@ -0,0 +1,5 @@
+package net.ripe.db.nrtm4.dao;
+
+public record ObjectData(int objectId, int sequenceId) {
+
```

Comment:
Can we rename it to something more meaningful like rpslObjectData? There is already RpslObjectInfo , may be we can re-use that if it fits the purpose?

Reply:
No its okey if it does not fit the bill